Firestore/core/src/api/load_bundle_task.h

```diff
@@ -143,7 +144,7 @@ class LoadBundleTask {
    * Removes the observer associated with the given handle, does nothing if the
    * callback cannot be found.
    */
-  void RemoveObserver(LoadBundleHandle);
+  void RemoveObserver(const LoadBundleHandle&);
```

Comment:
Nit: usually we don't omit parameter names in declarations (even though it's valid from the language point of view).

Reply:
Done.